packages/shared/lib/integrations/scripts/connection/linear-post-connection.ts

```diff
@@ -0,0 +1,25 @@
+import type { InternalNango as Nango } from './connection.manager.js';
+
+export default async function execute(nango: Nango) {
+    const query = `
+        query {
+            organization {
+                id
+            }
+        }`;
+
+    const response = await nango.proxy({
+        baseUrlOverride: 'https://api.linear.app',
```

Comment:
A `baseUrlOverride` isn't needed as this is the same already registered URL for `linear`: https://github.com/NangoHQ/nango/blob/65821af7922c78d1a1b1fdaea8fd13c37fcaeaf5/packages/shared/providers.yaml#L796